pyproject.toml

```diff
@@ -1,14 +1,15 @@
 [tool.towncrier]
-    package="web3"
-    filename = "docs/releases.rst"
-    directory = "newsfragments"
-    underlines = ["-", "~", "^"]
-    issue_format = "`#{issue} <https://github.com/ethereum/web3.py/issues/{issue}>`__"
-    title_format = "v{version} ({project_date})"
+# Read https://github.com/ethereum/web3.py/blob/main/newsfragments/README.md for instructions
+package = "web3"
+filename = "docs/release_notes.rst"
```

Comment:
This should remain `releases.rst` at least until we apply the full template to `web3.py`

Reply:
set back to `releases.rst`